app/views/catalog/_facet_pagination.html.erb

```diff
@@ -1,3 +1,13 @@
+    <% if @facet.index_pagination && params["facet.sort"] == 'index' %>
```

Comment:
Instead of checking the URL params directly, can we check if the facet field is sorted by index? Perhaps using:

https://github.com/projectblacklight/blacklight/blob/master/lib/blacklight/solr_response/facets.rb#L41-L43